content/strategy-goals/strategy/index.md

```diff
@@ -29,7 +29,7 @@ Our [values](../../company-info-and-process/values/index.md) are the principles
 
 We have identified five key use cases for our universal code search platform:
 
-1. Developer onboarding & velocity
+1. [Developer onboarding & velocity](use_cases/dev_velocity.md)
```

Comment:
Maybe include onboarding in URL for better search results? Not sure if it matters.

Reply:
```suggestion
1. [Developer onboarding & velocity](use_cases/dev-onboarding.md)
```